modules/Kconfig.tfm

```
@@ -84,17 +84,20 @@ config TFM_ISOLATION_LEVEL
 	  1,2 or 3; the default is set by build configuration.
 
 choice TFM_BL2
-        prompt "BL2 configuration, should TFM build with MCUboot support"
-        default TFM_BL2_CONFIG_FILE_DEFAULT
+	prompt "BL2 configuration, should TFM build with MCUboot support"
+	default TFM_BL2_CONFIG_FILE_DEFAULT
 
 config TFM_BL2_CONFIG_FILE_DEFAULT
-        bool "Use TFM BL2 setting from TFM configuration file"
+	bool "Use TFM BL2 setting from TFM configuration file"
+	help
+	  This option does not set -DBL2 in TFM, which means board's default
+	  is used. Currently all boards have BL2=ON as default.
```

TF-M uses the term `platforms`, so maybe it is better to use TF-M Platform instead of board ?
https://ci-builds.trustedfirmware.org/static-files/MBqpg7FOB3e4jpcJW8Nfq0G9cXO3vR0hJC0xDODqZnAxNjA2MzAwMzI4NTQ4Ojk6YW5vbnltb3VzOnZpZXcvVEYtTS9qb2IvdGYtbS1idWlsZC1kb2NzLW5pZ2h0bHkvbGFzdFN0YWJsZUJ1aWxkL2FydGlmYWN0/trusted-firmware-m/build/install/doc/user_guide/html/platform/ext/index.html#platforms

Also helps the reader to distinguish between a Zephyr board, and a TF-M platform.

```suggestion
	  This option does not set -DBL2 in TFM, which means the TF-M platform's default
	  is used. Currently all TF-M platforms have BL2=ON as default.
```